.circleci/config.yml

```
@@ -7,15 +7,15 @@ orbs:
 parameters:
   docker_image_tag:
     type: string
-    default: '7.6-node18-x86_64'
+    default: '7.7-node20-x86_64'
```

```suggestion
    default: '7.6-node20-x86_64'
```

since we are on 7.6.7